src/main/java/org/cbio/gdcpipeline/GDCPipelineApplication.java

```diff
@@ -42,7 +44,7 @@ private static void help(Options gnuOptions, int exitStatus, String opt) {
 		System.exit(exitStatus);
 	}
 
-	private static void launchJob(String[] args, String sourceDirectory, String outputDirectory, String study, String filter_sample) throws Exception {
+	private static void launchJob(String[] args, String sourceDirectory, String outputDirectory, String study, String filter_sample, String run_step) throws Exception {
```

Comment:
Hmm.. Isn't camel casing common though : https://stackoverflow.com/questions/414001/variable-naming-conventions-in-java